src/Neo.CLI/config.mainnet.json

```diff
@@ -37,7 +37,8 @@
     "Hardforks": {
       "HF_Aspidochelone": 1730000,
       "HF_Basilisk": 4120000,
-      "HF_Cockatrice": 5450000
+      "HF_Cockatrice": 5450000,
+      "HF_Domovoi": 5570000
```

Comment:
Could you please set the Domovoi height to 5800000 for NeoFS mainnet config? See the https://github.com/nspcc-dev/neo-go/pull/3487#discussion_r1635193295.

Reply:
Sure